src/app/extensions/quickorder/shared/direct-order/direct-order.component.spec.ts

```diff
@@ -53,7 +53,7 @@ describe('Direct Order Component', () => {
     fixture.detectChanges();
     expect(element.querySelectorAll('formly-group formly-field')).toMatchInlineSnapshot(`
       NodeList [
-        <formly-field
+        <formly-field _nghost-a-c51=""
```

Comment:
How about removing the `_nghost` [here](https://github.com/intershop/intershop-pwa/blob/256bd7f6690fd51a2f54096371ab0dc7c54d0519/src/jest-serializer/AngularHTMLSerializer.js#L13-L17)?

Reply:
@dhhyi Thank you very much for this tip. I had no idea regarding this possibility but the snapshot changes where distracting. 😄